pkg/preparer/setup.go

```diff
@@ -525,6 +538,17 @@ func New(preparerConfig *PreparerConfig, logger logging.Logger) (*Preparer, erro
 
 	podFactory := pods.NewFactory(preparerConfig.PodRoot, preparerConfig.NodeName, fetcher, preparerConfig.RequireFile)
 	podFactory.SetOSVersionDetector(osVersionDetector)
+
+	if preparerConfig.OpenContainerConfig.ExpectAtLeastOneBindMountEnvVar {
```

Comment:
Sadly this is necessary because I believe if the variable is renamed and the build flag isn't, the build will happily succeed but the variable won't be set! so we can set this bool in our config we expect there to be any "bind mount environment variables" and at least see a runtime error if the build flag didn't work right